test/bun.js/fs.test.js

```diff
@@ -420,6 +420,8 @@ describe("rm", () => {
     } catch (e) {}
     expect(existsSync(path)).toBe(true);
     rmSync(path);
+    expect(existsSync(path)).toBe(true);
+    rmSync(path, { recursive: true, force: true });
```

Comment:
With this change, the test now fails:
```
418 |     try {
419 |       mkdirSync(path);
420 |     } catch (e) {}
421 |     expect(existsSync(path)).toBe(true);
422 |     rmSync(path);
423 |     expect(existsSync(path)).toBe(true);
        ^
error: 
        Expected: true
        Received: false
      at /root/bun/test/bun.js/fs.test.js:423:4
✗ rm > removes a dir
```

Reply:
hmmm, strange. It is passing on my machine.
<img width="315" alt="Screenshot 2023-01-19 at 10 09 15 PM" src="https://user-images.githubusercontent.com/35280289/213629148-9534223a-3d20-4681-ae65-c7ed18d2fa27.png">

Reply:
I ran on a Linux box on the cloud:
```console
$ uname -a
Linux Ubuntu-2204-jammy-amd64-base 5.15.0-56-generic #62-Ubuntu SMP Tue Nov 22 19:54:14 UTC 2022 x86_64 x86_64 x86_64 GNU/Linux
```